cou/steps/execute.py

```diff
@@ -61,8 +60,8 @@ async def _apply_plan(plan: UpgradeStep, interactive: bool) -> None:
     :param interactive:
     :type interactive: bool
     """
-    result = "X"
-    while result.casefold() not in AVAILABLE_OPTIONS:
+    result = ""
+    while result == "" or result.casefold() not in AVAILABLE_OPTIONS:
```

Comment:
ah yes, because an empty string is considered to always be part of a string. You could (non blocking) define `AVAILABLE_OPTIONS` as a list and get rid of the `result == ""` comparison.

Reply:
done